### PR TITLE
Poll: replies should say 'timer', not 'timeout'

### PR DESCRIPTION
**Pull request by panpawn**
"Timeout" is mostly a programming term; not something I want to show users. - Zarel

### CODE REVIEW

File: chat-plugins/poll.js [hunk 1]
```diff
@@ -219,12 +219,12 @@ exports.commands = {
...
 				}), (timeout * 60000));
-				room.add("The timeout for the poll was set to " + timeout + " minutes.");
-				return this.privateModCommand("(The poll timeout was set to " + timeout + " minutes by " + user.name + ".)");
+				room.add("The timer for the poll was set to " + timeout + " minutes.");
+				return this.privateModCommand("(The poll timer was set to " + timeout + " minutes by " + user.name + ".)");
 			} else {
 				if (!this.canBroadcast()) return;
 				if (room.poll.timeout) {
-					return this.sendReply("The timeout for the poll is " + room.poll.timeoutMins + " minutes.");
+					return this.sendReply("The timer for the poll is " + room.poll.timeoutMins + " minutes.");
```
Comment (Zarel) [inline chat-plugins/poll.js]: Better, but I'd prefer something like:

"The poll will automatically end X minutes after it started."

File: chat-plugins/poll.js [hunk 2]
```diff
@@ -219,12 +219,12 @@ exports.commands = {
 					room.poll.end();
 					delete room.poll;
 				}), (timeout * 60000));
-				room.add("The timeout for the poll was set to " + timeout + " minutes.");
-				return this.privateModCommand("(The poll timeout was set to " + timeout + " minutes by " + user.name + ".)");
+				room.add("The timer for the poll was set to " + timeout + " minutes.");
```
Comment (Zarel) [inline chat-plugins/poll.js]: Something like

"The poll will automatically end in X minutes."

File: chat-plugins/poll.js [hunk 3]
```diff
@@ -209,7 +209,7 @@ exports.commands = {
 					clearTimeout(room.poll.timeout);
 					room.poll.timeout = null;
 					room.poll.timeoutMins = 0;
-					return this.add("The timeout for the poll was cleared.");
+					return this.add("The timer for the poll was cleared.");
```
Comment (Zarel) [inline chat-plugins/poll.js]: Perhaps "removed" would be a better term.

lol, programmers who just reuse the names of the actual functions they use.

Comment (bumbadadabum) [inline chat-plugins/poll.js]: Hey I like my function names!
yeah this is something I suck at so feel free to change it. I'm neither a
native speaker or good with words.

**Comment by panpawn** [comment]
Updated

File: chat-plugins/poll.js (hunk 3 above)
Comment (Zarel) [inline chat-plugins/poll.js]: Or "turned off"?

File: chat-plugins/poll.js [hunk 4]
```diff
@@ -209,7 +209,7 @@ exports.commands = {
 					clearTimeout(room.poll.timeout);
 					room.poll.timeout = null;
 					room.poll.timeoutMins = 0;
-					return this.add("The timeout for the poll was cleared.");
+					return this.add("The timer for the poll was removed.");
```
Comment (Zarel) [inline chat-plugins/poll.js]: Let's try "turned off"

Comment (Zarel) [inline chat-plugins/poll.js]: "The poll timer was turned off."

File: chat-plugins/poll.js [hunk 5]
```diff
@@ -219,12 +219,12 @@ exports.commands = {
 					room.poll.end();
 					delete room.poll;
 				}), (timeout * 60000));
-				room.add("The timeout for the poll was set to " + timeout + " minutes.");
-				return this.privateModCommand("(The poll timeout was set to " + timeout + " minutes by " + user.name + ".)");
+				room.add("The poll will automatically end in " + timeout + " minutes.");
```
Comment (Zarel) [inline chat-plugins/poll.js]: And correspondingly, "The poll timer was turned on: the poll will end in X minutes"

File: chat-plugins/poll.js [hunk 6]
```diff
@@ -219,12 +219,12 @@ exports.commands = {
...
+				return this.privateModCommand("(The poll timer was set to " + timeout + " minutes by " + user.name + ".)");
 			} else {
 				if (!this.canBroadcast()) return;
 				if (room.poll.timeout) {
-					return this.sendReply("The timeout for the poll is " + room.poll.timeoutMins + " minutes.");
+					return this.sendReply("The poll will automatically end " + room.poll.timeoutMins + " minutes after it started.");
```
Comment (Zarel) [inline chat-plugins/poll.js]: "The poll timer is on and set to X minutes"

File: chat-plugins/poll.js [hunk 7]
```diff
@@ -219,12 +219,12 @@ exports.commands = {
...
 				} else {
 					return this.sendReply("There's no timer for this poll.");
```
Comment (Zarel) [inline chat-plugins/poll.js]: "The poll timer is off."

**Comment by panpawn** [comment]
Re-updated

**Pull request merged**